### PR TITLE
add wikiCreationDate magic word (T6619)

### PR DESCRIPTION
**Pull request by Universal-Omega**

### CODE REVIEW

File: includes/WikiDiscover.php [hunk 1]
```diff
@@ -181,6 +182,23 @@ public static function numberOfWikisBySetting( $parser, $setting = null, $value
...
 	
+	/**
+	 * @param Parser $parser
+	 * @param string $wikiDatebase|null
+	 * @return string
+	 */
+	public static function wikiCreationDate( Parser $parser, String $wikiDatabase = null ) {
+		$config = MediaWikiServices::getInstance()->getMainConfig();
+
+		$dbw = wfGetDB( DB_MASTER, [], $config->get( 'CreateWikiDatabase' ) );
```
Comment (JohnFLewis) [inline includes/WikiDiscover.php:193]: why DB_MASTER?

Comment (Universal-Omega) [reply includes/WikiDiscover.php]: Switched to DB_REPLICA

File: includes/WikiDiscover.php [hunk 2]
```diff
@@ -115,32 +115,33 @@ public static function onParserFirstCallInit( Parser $parser ) {
...
 	}
 	
 	/**
 	 * @param Parser $parser
-	 * @param string $category|null
+	 * @param string $category|uncategorized
 	 * @return integer
 	 */
-	public static function numberOfWikisInCategory( Parser $parser, String $category = null ) {
+	public static function numberOfWikisInCategory( Parser $parser, String $category = 'uncategorized' ) {
```
Comment (JohnFLewis) [inline includes/WikiDiscover.php:126]: uncategorised

File: includes/WikiDiscover.php [hunk 3]
```diff
@@ -181,6 +182,23 @@ public static function numberOfWikisBySetting( $parser, $setting = null, $value
...
+	 * @param Parser $parser
+	 * @param string $wikiDatebase|null
+	 * @return string
+	 */
+	public static function wikiCreationDate( Parser $parser, String $wikiDatabase = null ) {
+		$config = MediaWikiServices::getInstance()->getMainConfig();
+
+		$dbr = wfGetDB( DB_REPLICA, [], $config->get( 'CreateWikiDatabase' ) );
+		
+		$wikiDatabase = $wikiDatabase ? $wikiDatabase : $config->get( 'DBname' );
```
Comment (JohnFLewis) [inline includes/WikiDiscover.php:195]: $wikiDatabase = $wikiDatabase ?? $config->get( 'DBname' );

File: includes/WikiDiscover.php (hunk 2 above)
Comment (Universal-Omega) [reply includes/WikiDiscover.php]: Fixed

File: includes/WikiDiscover.php (hunk 3 above)
Comment (Universal-Omega) [reply includes/WikiDiscover.php]: Fixed

**Pull request merged**